ddtrace/contrib/django/apps.py

```diff
@@ -49,17 +49,20 @@ def ready(self):
         if settings.AUTO_INSTRUMENT:
             # trace Django internals
             insert_exception_middleware()
-            try:
-                patch_db(tracer)
-            except Exception:
-                log.exception('error patching Django database connections')
 
             try:
                 patch_template(tracer)
             except Exception:
                 log.exception('error patching Django template rendering')
 
-            try:
-                patch_cache(tracer)
-            except Exception:
-                log.exception('error patching Django cache')
+            if settings.DATABASE_AUTO_INSTRUMENT:
```

Comment:
Side note for the future: I'd rather prefer a different approach instead of multiple `if` blocks, but since we don't provide a different way to patch Django, that one is probably the best  we can have now.